services/lumberjack/src/router.js

```diff
@@ -0,0 +1,13 @@
+import koaProtobuf from 'koa-protobuf';
```

Comment:
Honestly, this service doesn't need an API at all (at least for now). You can just remove the router, and `PORT` env var and all the associated dependencies.